conf/app.routes

```diff
@@ -141,3 +141,6 @@ POST        /furlough-in-last-tax-year                       controllers.Furloug
 
 GET         /multiple-furlough-unsupported                   controllers.CalculationUnsupportedController.multipleFurloughUnsupported()
 GET         /start-date-within-lookback-unsupported          controllers.CalculationUnsupportedController.startDateWithinLookbackUnsupported()
+
+GET        /employeeFirstFurloughed                              controllers.EmployeeFirstFurloughedController.onPageLoad()
```

Comment:
urls are usually hyphenated. Might want to drop Georgie a message if there isn't one on the ticket/slides